src/main/resources/swagger/schemas/matches-json-schema-pattern.yaml

```diff
@@ -3,23 +3,40 @@ type: object
 properties:
   matchesJsonSchema:
     oneOf:
-      - type: string
-        example: "//Order/Amount"
       - type: object
-        allOf:
-          - properties:
-              expression:
-                type: string
-                example: "//Order/Amount"
-          - $ref: "content-pattern.yaml"
+        description: A valid JSON schema object
+        example:
+          type: object
+          required:
+            - name
+          properties:
+            name:
+              type: string
+            tag:
+              type: string
 
-        required:
-          - expression
+      - type: string
+        description: A valid string containing a JSON-encoded JSON schema
+        example: |-
+          {
+            "type": "object",
+            "required": [
+              "name"
+            ],
+            "properties": {
+              "name": {
+                "type": "string"
+              },
+              "tag": {
+                "type": "string"
+              }
+            }
+          }
 
-  xPathNamespaces:
-    type: object
-    additionalProperties:
-      type: string
+  schemaVersion:
```

Comment:
This could be an enum of the following values:
V4
V6
V7
V201909
V202012

Reply:
Good shout, updated with the change